From: Yavor Doganov <yavor@gnu.org>
Date: Wed, 10 Oct 2018 18:26:12 +0300
Subject: configure: set foreign

GNU expects a README but we don't have one.
So by setting the foreign flag, we fix autoreconf

Bug-Debian: https://bugs.debian.org/765214
---
 configure.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.in b/configure.in
index aec96d4..fde3865 100644
--- a/configure.in
+++ b/configure.in
@@ -5,6 +5,7 @@ GJITEN_VERSION=2.6
 AC_INIT(gjiten, 2.6)
 AC_CONFIG_SRCDIR(src/gjiten.c)
 AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
+AM_INIT_AUTOMAKE([foreign])
 
 dnl AM_MAINTAINER_MODE
 AM_CONFIG_HEADER(config.h)
